llpc/tool/amdllpc.cpp

```diff
@@ -232,10 +232,14 @@ static cl::opt<bool> EnableRelocatableShaderElf("enable-relocatable-shader-elf",
                                                 cl::desc("Compile pipelines using relocatable shader elf"),
                                                 cl::init(false));
 
-// -check-auto-layout-compatible: check if auto descriptor layout got from spv file is commpatible with real layout
+// -check-auto-layout-compatible: check if auto descriptor layout got from spv file is compatible with real layout
 static cl::opt<bool> CheckAutoLayoutCompatible(
     "check-auto-layout-compatible",
-    cl::desc("check if auto descriptor layout got from spv file is commpatible with real layout"));
+    cl::desc("Check if auto descriptor layout got from spv file is compatible with real layout"));
+
+// -scratch-bounds-checks: insert scratch access bounds checks on loads and stores
+static cl::opt<bool> EnableScratchAccessBoundsChecks(
+    "scratch-bounds-checks", cl::desc("Insert scratch access bounds checks on loads and stores"), cl::init(false));
```

Comment:
scratch-bounds-checks -> enable-scratch-access-bound-checks
Keep the option name consistent with the variable name